packages/features/webhooks/lib/handleWebhookScheduledTriggers.ts

```diff
@@ -52,17 +52,25 @@ export async function handleWebhookScheduledTriggers(prisma: PrismaClient) {
     }
 
     const headers: Record<string, string> = {
-      "Content-Type":
-        !job.payload || jsonParse(job.payload) ? "application/json" : "application/x-www-form-urlencoded",
+      "Content-Type": "application/json",
     };
 
+    const parsedPayload = jsonParse(job.payload);
+
+    const body = JSON.stringify({
```

Comment:
I think we should be able to use `sendPayload` that would handle everything including headers, body, secret and the fetch request